src/dune_rules/coq_rules.ml

```diff
@@ -370,27 +370,31 @@ module Context = struct
     { t with boot_type }
 end
 
-let parse_coqdep ~dir ~(boot_type : Bootstrap.t) ~coq_module
-    (lines : string list) =
+let parse_coqdep ~dir ~(boot_type : Bootstrap.t) ~coq_module contents =
```

Comment:
`lines` is a better name than `contents`, moreover when this is called from `lines_of`